src/tictactoe/Human.java

```diff
@@ -0,0 +1,11 @@
+package tictactoe;
+
+public class Human extends Player {
+    public Human(Marker marker) {
+        super(marker);
+    }
+
+    public Turn take_turn(Space space) {
```

Comment:
in java world all methods are camelCased.

Reply:
:+1: